cmd/algocfg/profileCommand.go

```diff
@@ -69,7 +69,9 @@ var (
 	relay = configUpdater{
 		description: "Relay consensus messages across the network and support catchup.",
 		updateFunc: func(cfg config.Local) config.Local {
-			cfg.Archival = true
+			cfg.MaxBlockHistoryLookback = 20000
+			cfg.CatchpointFileHistoryLength = 3
+			cfg.CatchpointTracking = 2
```

Comment:
maybe also `CatchpointInterval = 10000` ?

Reply:
The default is 10000 right now, I was hesitant to introduce an explicit override in case we want the ability to more easily change later.

Reply:
+1, would leave to config defaults.

Reply:
This is interesting - CatchpointTracking was 0 by default, which would enable it on archivals before; setting to 2 explicltly says catchpoint tracking + catchpoint file generation? I think I usually set this to 1 for labels (no files) only for validation on non-archivals, so 2 is new to me.

Reply:
Confirmed with @algorandskiy the CatchpointTracking setting as well.